stories/tsconfig.json

```diff
@@ -3,6 +3,7 @@
     "extends": "../tsconfig.json",
     "include": ["../src/typings.d.ts", "./*.ts", "./*.tsx"],
     "exclude": ["src/**/*.spec.ts"],
+    "files": [""],
```

Comment:
what is the purpose? :)

Reply:
Yes, this is some kind of bug, when `files` is empty it sometimes fails to build for some reason. This fixes problem